### PR TITLE
Horizontal and vertical scroll bars on the Kiosk

### PR DESCRIPTION
**Pull request by andreeaceachir142**
# What 

- This PR is a combination of both the vertical and horizontal scroll bars

### CODE REVIEW

File: packages/components/src/components/scroll-buttons/test.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
 </head>
 
 <body>
-    <section style="display: flex;">
+    <section style="display: flex; background-color: #eee;">
```
Comment (andreeaceachir142) [inline packages/components/src/components/scroll-buttons/test.html:21]: Do we need this change here? 🤔

Comment (ammapspeople) [reply packages/components/src/components/scroll-buttons/test.html:21]: For internal testing. The buttons are now white, which makes them hard to see on a white background.

**Comment by andreeaceachir142** [comment]
@ammapspeople Added new icons, removed the scroll bar and aligned the buttons with the categories. Ready for re review 🙏🏻

File: packages/map-template/src/components/Search/Categories/Categories.scss [hunk 2]
```diff
@@ -3,16 +3,21 @@
     flex-shrink: 0;
     display: grid;
     grid-auto-flow: column;
-    align-items: end;
     gap: var(--spacing-x-small);
+    margin-top: var(--spacing-x-small);
 
+    ::-webkit-scrollbar {
```
Comment (ammapspeople) [inline packages/map-template/src/components/Search/Categories/Categories.scss:9]: Will this work in Firefox and Safari?

Comment (ammapspeople) [reply packages/map-template/src/components/Search/Categories/Categories.scss:9]: I may just answer this myself: It does not work in Firefox 😄

Comment (andreeaceachir142) [reply packages/map-template/src/components/Search/Categories/Categories.scss:9]: Can you send me a screenshot of how it looks on Firefox please? 🙏🏻

Comment (ammapspeople) [reply packages/map-template/src/components/Search/Categories/Categories.scss:9]: No, because you fixed it 😄

**Review by ammapspeople — COMMENTED** [review]
👍

**Pull request merged**